src/steps/preprocessing/misc.py

```diff
@@ -3,7 +3,7 @@
 from sklearn.externals import joblib
 from sklearn.feature_extraction import text
 
-from steps.base import BaseTransformer
+from ...steps.base import BaseTransformer
```

Comment:
@taraspiotr I didn't know about the  ... concept.
Now that I did a quick search this came up:

` However one of the guys running the workshop said that we should use no more than two dots and Guido van Rossum would personally chop off the fingers of those who use triple dot notation for relative imports. I always recall this when it comes to triple dot notation so I avoid to use it.`

So I'd rather not use it and think about what can be done about it. Sklearn deals with similar problems by putting stuff in `__init__.py` https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/ensemble/__init__.py